fdi/language/tr/FDSETUP.DEF

```diff
@@ -82,7 +81,7 @@ GATHERING="Kurulum haz
 # STAGE800 - Installing
 INSTALL_FRAME=/w62 /h10 /c
 INSTALL_OPTS=/w48 /h5 /c
-INSTALL=/s- /f %1 "%2" /f %3 " art�k kurulabilir."
+INSTALL=/s- /f %1 "%2" /f %3 " art�k kurulabilir.

```

Comment:
Missing end quote.

Reply:
Now this is an interesting string. Actually it should be like this innit:

```
INSTALL=/s- /f %1 '%2' /f %3 "art�k kurulabilir."
```

The space before `artık` seems unnecessary. I changed the original double quotes around `%2` to single quotes, to make it more understandable, that it is surrounded by literal quotes.

Reply:
> The space before `artık` seems unnecessary. 

Well that's a tricky one. :-)

Let's assume %2 equals `FreeDOS`

The `/s-` turns off spacing between parameters. So, `/s- /f %1 '%2' /f %3 "art�k kurulabilir."` would display `FreeDOSart�k kurulabilir.` because no space will be inserted between the `'%2'` and the following text string.

So the space after the quote forces one. Don't know why a `/s-` was placed in there. Looking at the BAT file that uses that string it does not modify the spacing behaviour. 

`vecho /k0 /t %FLANG% INSTALL %TFH% "%OS_NAME% %OS_VERSION%" %TFF%`
_(**FLANG** is the language file, **TFH** and **TFF** are theme color values and **OS_NAME** + **OS_VERSION** are in quotes and passed as a single parameter.)_

Possibly, a long time ago that invocation turn off spacing and the `/s-` in the translation file was just a reminder of the state. However, it would have been better to use `/s+` to turn back on spacing. 

So, remove the `/s-` from the string and the space following the quote won't be needed. 

An example of `/s-` behaviour...

`vecho A B /s- C  D     E "F G" "H" /s+ I     J "K    L"    "M N"` displays `A BCDEF GH I J K    L M N`